db/fixtures/miq_product_features.yml

```diff
@@ -5123,6 +5123,54 @@
           :description: Edit Job Templates Tags
           :feature_type: control
           :identifier: configuration_script_tag
+
+# Embedded Automation Manager
+- :name: Embedded Automation Manager
+  :description: Everything under Ansible
```

Comment:
@Fryguy @dclarizio - the menu says Ansible for the internal one. What should it be here?